sensorprocess/sensorprocess.go

```diff
@@ -30,13 +30,14 @@ type Config struct {
 func Start(
 	ctx context.Context,
 	config Config,
+	isOnline bool,
```

Comment:
Should we call this `isOnline` or `isLive`? 

To me, "online" is the opposite of "offline" and "live" is the opposite of a  "replay sensor" but happy to change if live makes more sense

Reply:
(see other comment)